keen/explorer
# Alert users when there are no collections in preview collections modal

## What does this PR do? How does it affect users?

Resolves https://github.com/keen/explorer/issues/164. 

The "Preview Collections" modal now displays a notification to users when there are no collections in the project.

![screen shot 2016-07-15 at 4 57 20 pm](https://cloud.githubusercontent.com/assets/6183404/16891350/416e8b4e-4aad-11e6-9bb4-727d99c92375.png)

## How should this be tested?

- Get a sample project with no collections running locally. 
- Click "Preview Collections" and ensure the modal pictured above is displayed. 
- Confirm that collections are displayed in the modal when collections are added to the project.
